tests/console/kdump_and_crash.pm

```diff
@@ -15,12 +15,17 @@ use strict;
 use testapi;
 use utils;
 use kdump_utils;
+use registration;
+use version_utils;
 
 sub run {
     my ($self) = @_;
     select_console('root-console');
 
     # preparation for crash test
+    if (sle_version_at_least('15')) {
```

Comment:
To be sure to execute this code only on SLE, you need to also call `is_sle` (it is not part of `sle_version_at_least`).